docs/configuration.adoc

```diff
@@ -195,183 +195,219 @@ The following script levels are reserved by pre-defined scripts:
-<57> The cache size in MB. This is distributed among various caches used in DocumentNodeStore.
-<58> Percentage of cache to be allocated towards the Node cache.
-<59> Percentage of cache to be allocated towards the Previous Document cache.
-<60> Percentage of cache to be allocated towards the Children cache.
-<61> Percentage of cache to be allocated towards the Diff cache.
-<62> The number of segments in the LIRS cache (default 16, a higher count means higher concurrency but slightly lower cache hit rate).
-<63> The delay to move entries to the head of the queue in the LIRS cache (default 16, a higher value means higher concurrency but slightly lower cache hit rate).
-<64> Flag that indicates if Node bundling is disabled.
-<65> Flag indicating if external changes should be pre-fetched in a background thread.
-<66> Number of content updates that need to happen before the updates are automatically purged to the private branch.
-<67> The max age (in milliseconds) that journal (for external changes) entries are kept (older ones are candidates for gc).
-<68> Paths which should be cached in persistent cache.
-<69> The path to the directory where the persistent cache will be stored.
-<70> The path to the directory where the persistent journal cache will be stored.
-<71> The configuration for a Mongo DB node document store. If used, the top-level `mongo` properties must also be configured.
-<72> Enables the use of a Mongo database for the document store.
-<73> Value in seconds. Determines the duration beyond which it can be safely assumed that the state on the secondaries is consistent with the primary, and it is safe to read from them.
-<74> The configuration for a relational database node store. The database configured in the `dbProp.properties` file will be used.
-<75> Enables the use of a relational database for the document node store.
-<76> The configuration for the Mongo DB connection.
-<77> The Mongo DB hostname and ports.
-<78> The name of the database. Required if Mongo is used.
-<79> The username used for authentication.
-<80> The password used for authentication.
-<81> The authentication database, if different from the storage database.
-<82> The required replica set name.
-<83> Flag that indicates if an SSL connection should be used.
-<84> Flag that indicates if socket keep-alive should be enabled for connections to MongoDB.
\ No newline at end of file
+<1> The Key-Value Storage type.
+<2> The configuration for a MapDB key-value storage.
+<3> The directory for a MapDB key-value storage.
+<4> The configuration for the blob store.
+<5> The type of blob store. NOTE: The local blob storage should only be used with a single server and single scheduler running on the same machine.
+<6> Configuration for a file-based blob store.
+<7> The directory where the blobs are created.
+<8> Example configuration for an AWS cloud-native implementation. AWS uses DynamoDB for key-value storage and S3 for blob storage.
```

Comment:
Should also mention that this is for S3-compatible object stores, like MinIO and Ceph.

Reply:
This should mention that this is only used to create the table if it does not already exist.
You also need to document the boolean `onDemandCapacity` option which, when set to true, uses on-demand capacity instead of provisioned capacity when creating the table.